### PR TITLE
Test mode to run unit tests 

### PR DESCRIPTION
**Pull request by oskardrenske-ith**

### CODE REVIEW

File: sokannonser/repository/text_to_concept_methods.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import re
+
+RE_PLUS_MINUS = re.compile(r"((^| )[+-])", re.UNICODE)
```
Comment (halge) [inline sokannonser/repository/text_to_concept_methods.py:3]: can you add a comment what does it do?

File: sokannonser/repository/text_to_concept.py [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
 from sokannonser.repository.elastic_connection_with_retries import create_elastic_client_with_retry
-
+from sokannonser.repository.text_to_concept_methods import clean_plus_minus
 log = logging.getLogger(__name__)
 
 OP_NONE = ''
 OP_PLUS = '+'
 OP_MINUS = '-'
-
+RE_PLUS_MINUS = re.compile(r"((^| )[+-])", re.UNICODE)
```
Comment (halge) [inline sokannonser/repository/text_to_concept.py:12]: can you add a comment what does it do?

File: tests/test_resources/helper.py [hunk 3]
```diff
@@ -246,3 +247,9 @@ def check_ads_for_country_in_address(hits, abroad):
         assert (country == 'Sverige') != abroad
         assert (country_concept_id == 'i46j_HmG_v64') != abroad
 
+
+def is_dst():
```
Comment (halge) [inline tests/test_resources/helper.py:251]: there is no dst anymore in Sweden, remove.

File: tests/unit_tests/test_marshal_platsannons_results_integration.py [hunk 4]
```diff
@@ -90,7 +87,8 @@ def test_values_marshal_mocked_elastic_result():
     args = {settings.FREETEXT_QUERY: False, settings.STATISTICS: False}
 
     pbsearch = Search()
-    querybuilder = QueryBuilder(mock.MockTextToConcept)
+    settings.TEST_MODE = True
```
Comment (halge) [inline tests/unit_tests/test_marshal_platsannons_results_integration.py:90]: why doubling(see up, line 24-26) ?

File: tests/unit_tests/test_querybuilder.py [hunk 5]
```diff
@@ -90,6 +91,7 @@ def test_parse_args_query_with_slash():
             'position': None, 'position.radius': None, 'employer': None, 'q': 'systemutvecklare/programmerare',
             'qfields': None, 'relevance-threshold': None, 'sort': None, 'stats': None, 'stats.limit': None}
 
+    expected_time = 'now+2H/m' if is_dst() else 'now+1H/m'  # f"{expected_time}"
```
Comment (halge) [inline tests/unit_tests/test_querybuilder.py:94]: remove if, else

File: tests/unit_tests/test_querybuilder.py [hunk 6]
```diff
@@ -6,6 +6,7 @@
 from sokannonser.repository import taxonomy
 from sokannonser.repository.querybuilder import QueryBuilder
 from tests.integration_tests.test_resources import mock_for_querybuilder_tests as mock
+from tests.test_resources.helper import is_dst
```
Comment (halge) [inline tests/unit_tests/test_querybuilder.py:9]: no need.

File: tests/unit_tests/test_querybuilder_experience.py [hunk 7]
```diff
@@ -15,10 +16,13 @@ def test_experience_values(value):
     else:
         value_in_expected = False
 
+    expected_time = 'now+2H/m' if is_dst() else 'now+1H/m'  # f"{expected_time}"
```
Comment (halge) [inline tests/unit_tests/test_querybuilder_experience.py:19]: remove if, else

**Review by halge — CHANGES_REQUESTED** [review]

File: sokannonser/repository/text_to_concept.py (hunk 2 above)
Comment (oskardrenske-ith) [reply sokannonser/repository/text_to_concept.py]: duplicated, removed

File: tests/unit_tests/test_querybuilder_experience.py (hunk 7 above)
Comment (oskardrenske-ith) [reply tests/unit_tests/test_querybuilder_experience.py:19]: then it'll break in October :)

File: tests/test_resources/helper.py (hunk 3 above)
Comment (oskardrenske-ith) [reply tests/test_resources/helper.py:251]: No, that decision hasn't been made yet.

File: tests/unit_tests/test_querybuilder.py (hunk 5 above)
Comment (oskardrenske-ith) [reply tests/unit_tests/test_querybuilder.py:94]: No decision on DST / Normal time has been made

**Review by halge — APPROVED** [review]

**Review by halge — APPROVED** [review]

**Review by lina1101 — APPROVED** [review]

**Pull request merged**